src/perlDebug.ts

```diff
@@ -559,7 +559,7 @@ class PerlDebugSession extends LoggingDebugSession {
 			.then(variables => {
 				const result = [];
 
-				if (id != null && variables[id]) {
+				if (id !== null && variables[id]) {
```

Comment:
yeah - it's a bit wierd, but it should be `!=` (without the type)
javascript have both `null` and `undefined` in this case we should catch both